reactive/jenkins.py

```diff
@@ -230,3 +248,34 @@ def update_nrpe_config(nagios):
 def stop():
     service_stop("jenkins")
     status_set("maintenance", "Jenkins stopped")
+
+
+@when("reset.jenkins.storage")
+def reset_jenkins_storage():
+    status_set("maintenance", "Configuring Jenkins Storage")
+
+    jenkins_installed = get_state("apt.installed.jenkins")
+    if jenkins_installed:
+        service_stop('jenkins')
```

Comment:
And for clarity, in case we copy the data over we could still use the symlink, so there would be no need to rewrite files in /etc/default or anything.